subjects/mobile-dev/bizz-card/README.md

```diff
@@ -1,81 +1,88 @@
 
-Hint: You can see some Flutter samples [here](https://flutter.github.io/samples/#?type=cookbook) 👩🏽‍🍳
+You can see some `Flutter` samples [here](https://flutter.github.io/samples/#?type=cookbook) 👩🏽‍🍳
 
-### Objective
-
-### Making your first app
+### Instructions
 
 BizzCard is a simple application which shows a static card with your personal information.
 
-This subject is divided into 3 parts. Overall objective is for you to learn:
-
-- About and apply the essence of Flutter - widgets.
-- Basic structure of a Flutter app.
-- How to run Flutter app on physical device or Android/iOS emulator.
-
-- Note: only standard Flutter packages and url_launcher (for bonus) are allowed
+This subject is divided into 3 parts. Overall objective is for you to learn about:
 
-### First Part
+- Applying the essence of `Flutter` - `widgets`.
+- Basic structure of a `Flutter` app.
+- How to run `Flutter` on physical devices or Android/iOS emulator.
 
-In this part:
+> Note: only standard `Flutter` packages and `url_launcher` (for bonus) are allowed.
 
-- Run Flutter generated counter app
-- Understand structure of the Flutter app
+**_First Part_**
 
-To create your first Flutter application open Android Studio or similar IDE and follow the steps:
+To create your first `Flutter` application open Android Studio or a similar IDE and follow the steps:
 
 1. Open the IDE and select **Start a new Flutter project**.
 2. Select **Flutter Application** as the project type. Then click **Next**.
-3. Specify path to **Flutter SDK’s location**
-   (select **Install SDK…** if the text field is blank).
+3. Specify the path to **Flutter SDK’s location** (select **Install SDK…** if the text field is blank).
 4. Enter the project name (for example, `myapp`). Then click **Next**.
 5. Click **Finish**.
 6. Wait for Android Studio to install the SDK and create the project.
 
-See the Run the app section [here](https://flutter.dev/docs/get-started/test-drive?tab=androidstudio#create-app) to run a sample app.
+See the `Run the app` section [here](https://flutter.dev/docs/get-started/test-drive?tab=androidstudio#create-app) to run a sample app.
 
-The starter point in Flutter app is in lib/main.dart. Change this file to change app's behavior.
+The starter point in Flutter app is in `lib/main.dart`. Change this file to change the app's behavior.
 
-### Second Part:
+**_Second Part:_**
 
-In this part:
+You must display the following information in the BizzCard:
 
-- Build your first app
+- First Name
+- Last Name
+- Age
+- Phone number
+- Email
+- Photo/Image
 
-You should display static information about yourself, i.e. name, surname, age, phone number, email, and image.
-Try to make it as it is shown in the example below:
+Here is an example of what we are asking:
 
 <!-- <center> -->
 <center>
 <img src="./resources/bizzCard.01.png?raw=true" style = "width: 210px !important; height: 420px !important;"/>
```

Comment:
In the https://preview.01-edu.org/preview the image is not rendered - at least not in my current browser (Brave - Chromium to 113.0.5672.92.)

Reply:
on my side its fine... not sure what is happening...
![image](https://github.com/01-edu/public/assets/53977751/c0701536-1811-401e-99dc-b1f9cc0466a7)